styleguide/sections/Overlays.js

```diff
@@ -63,34 +63,46 @@ export default class Overlays extends Component {
         <p>
           Very similar to TooltipToggle, but the children prop is the content you want inside the dropdown, and you pass in a trigger to open and close the dropdown.
         </p>
-        <div style={{textAlign: 'center'}}>
-          <Dropdown trigger={
-            <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
-          }>
+        <div style={{ textAlign: 'center' }}>
+          <Dropdown trigger=
```

Comment:
I did have

```jsx
<Dropdown trigger={
  <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
}
>
```
which doesn't look right. Then I tried:

```jsx
<Dropdown trigger={
  <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
}>
```
but it breaks the listing rules.

Changing to:
```jsx
<Dropdown
  trigger={
    <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
  }
>
```